### ISSUE TITLE
Breaks on new heroku buildpack v46

### ISSUE THREAD

**Issue by mboersma** [deis/example-java-jetty#6]
I tested updating Heroku's [Java buildpack v46](https://github.com/heroku/heroku-buildpack-java/releases/tag/v46) from v44 and found our example-java-jetty app fails with a NPE--something about parsing `[]`...

Probably this app needs some TLC to work with more modern Java libs / building.

**Comment by Joshua-Anderson** [comment]
Explicit error:

```
-----> Discovering process types
       Procfile declares types -> web
-e:1:in `<main>': undefined method `[]' for nil:NilClass (NoMethodError)
remote: 2016/08/26 00:42:33 Error running git receive hook [Build pod exited with code 1, stopping build.]
```

What odd is that it happens **after** successfully compiling. Need to look and find out what the buildpack is doing there.

**Comment by Joshua-Anderson** [comment]
`v45` is fine, so the problem is [here](https://github.com/heroku/heroku-buildpack-java/compare/v45...v46)

**Comment by Joshua-Anderson** [comment]
This is actually a bug in slugbuilder, which crashes on the output off `---` from the `bin/release` stage.

See deis/slugbuilder#103

**Comment by mboersma** [comment]
Fixed in deis/slugbuilder#103.
